src/V3LinkParse.cpp

```diff
@@ -693,6 +696,20 @@ class LinkParseVisitor final : public VNVisitor {
             iterateChildren(nodep);
         }
     }
+    void visit(AstCell* nodep) override {
+        if (nodep->origName().empty()) {
+            if (!VN_IS(nodep->modp(), Primitive)) {  // Module/Program/Iface
+                nodep->modNameFileline()->v3error("Instance of " << nodep->modp()->verilogKwd()
+                                                                 << " must be named");
+            }
+            // UDPs can have empty instance names. Assigning unique names for them to prevent any
+            // conflicts
+            const string newName = "$unnamedudp" + cvtToStr(++m_anonUdpId);
+            UINFO(0, nodep << newName << endl);
```

Comment:
```suggestion
```

Or make level 9 with appropriate commentary.  When you do make a temp UINFO(0, please put in a FIXME so we don't commit by mistake, thanks.

Reply:
Oh sorry forgot to remove it